test/integration/integration_test.go

```diff
@@ -102,6 +113,25 @@ func (i *integrationTest) setup() {
 		}
 		i.setupComplete = true
 	}
+
+	from := flag.CommandLine
+	if vflag := from.Lookup("v"); vflag != nil {
+		fmt.Fprint(os.Stdout, "integration_test.go: found -v flag from starter bash script with initial glog level ", getLogLevel(), "\n")
```

Comment:
OK ... I so far have not noticed the difference, and I liked the ability to choose between os.Stdout and os.Stderr.  I'll ponder/consider, pending additional reasons to switch.